tox.ini

```diff
@@ -60,7 +60,7 @@ commands =
     pip-audit {env:EXCLUDES:}
 
 [testenv:docs,docs-py{310,311,312}]
-description = Build documentation, including the examples, and check links
+description = Build documentation, passing posargs to control what should be built
```

Comment:
@Scienfitz @AdrianSosic just FYI: I would wait with this PR until Martin had the chance to also test this workflow locally since there is no need to rush here.

Reply:
what exactly should I test? just rebuilding the doc locally fully from scratch on my machine?

Reply:
ok so the build via `tox -e docs-py310` went through in a clean new dev env on main, am I looking for anything else?

Reply:
No, that is all this PR should enable - it basically removes the need to use the "-f" that Adrian still had. So I now interpret this ready for merge and will see how to reasonably merge this with regards to other current documentation PRs